### ISSUE TITLE
Disrupting toll-like receptor signaling--notes on annotation

### ISSUE THREAD

**Issue by genegodbold** [jhuapl-bio/pathogenesis-gene-ontology#166]
So this maybe just an FYI for these entities: PATHGO_0000103 (host toll-like receptor signaling disruption factor) AND PATHGO_0000233 (disrupts host toll-like receptor signaling)

There are dozens of these things in bacteria alone, particularly in the genus Mycobacterium. In my annotations I have been including proteins that:

1. Agonize toll-like receptors
2. Antagonize toll-like receptors
3. Interfere with intracelllular toll-like receptor signaling at the level of MyD88 or above (there are a few proteins that specifically interact with MyD88 to put the kibosh and TLR signaling in the cell.

Eventually we might want to distinguish the ones that are acting intracellularly versus the ones that are binding to the receptors extracellularly. Also, there may eventually be enough annotated to justify specifying *which* toll-like receptor is involved (TLR2 and TLR4 are the ones that seem to be getting hit the  most often).

## Attribution
If you would like a nanoattribution, please indicate your ORCID id.

**Comment by jacakrj1** [comment]
Added these notes to the editorialNote on the 'disrupts host toll-like receptor signaling'.  I think the 'agonist' and 'antagonist' concepts may be covered by the GO terms "GO:0034123 positive regulation of toll-like receptor signaling pathway" and "GO:0034122 negative regulation of toll-like receptor signaling pathway".  Thoughts?

**Comment by genegodbold** [comment]
Yes, but those terms (GO:0034123 and GO:0034122) are going to be "contaminated" with normal agonism/antagonism processes. The PATHGO term is for exogenous (non-native) modulation. And not passive (LPS binding to TLR4) but active. Is there a way to capture that sort of thing?
